pkg/adaptor/elasticsearch/elasticsearch.go

```diff
@@ -12,9 +12,9 @@ import (
 	elastigo "github.com/mattbaird/elastigo/lib"
 )
 
-// Elasticsearch is an adaptor to connect a pipeline to
+// ElasticSearch is an adaptor to connect a pipeline to
 // an elasticsearch cluster.
-type Elasticsearch struct {
+type ElasticSearch struct {
```

Comment:
this one might get people in a tizzy
the name is actually Elasticsearch, not ElasticSearch